docs/source/devguide/documentation.rst

```diff
@@ -64,87 +31,77 @@ Writing New Documentation
 
 -------------------------------------
 Version Control for New Documentation
 -------------------------------------
 
-The :file:`configure.sh` script automatically adds your newly generated HTML files and image files to Git for tracking of the ``gh-pages`` branch using the following console commands:
+With each new release, the website documentation will be updated by one of the core developers with the latest documentation from the **develop** branch. In order to **expose** newly generated HTML documentation to GitHub such that it will present on the website, the Sphinx and Doxygen build files must be copied to the **gh-pages** branch. Once the documentation is ready to be copied over to the **gh-pages** branch, the Sphinx and Doxygen HTML files need to be built:
 
 .. code-block:: none
 
-    cd sphinx
-    git add -A
-    git add -f _images/*
-    git add -f _images/math*
-    git add -f doxygen/html/* .
+    cd docs
+    make html
+    cd doxygen
+    doxygen Doxyfile
+    cd ../..
 
-You may commit your changes locally for the ``gh-pages`` branch just as you normally would expect to using Git:
+Now that the HTML files have been built, you can checkout the **gh-pages** branch and then checkout a new branch to store the new version of the documentation:
 
 .. code-block:: none
 
-    git commit -m 'my commit message here'
+    git checkout gh-pages
+    git checkout -b gh-pages-update
 
-Likewise, to push the changes made to your personal forked repository of OpenMOC, use the following command in the console:
+You can then copy the files from the :file:`docs` directory to the root OpenMOC directory on your new branch:
 
 .. code-block:: none
 
-    git push
-
-In order to submit new and/or revised documentation to the main OpenMOC reposotiry, please use a `pull request`_ to the ``gh-pages`` branch of the OpenMOC repository on GitHub as described in :ref:`Development Workflow <work_flow>`.
+    cp -r docs/build/html/* .
+    cp -r docs/doxygen/html doxygen/
 
+Since the :file:`.gitignore` file does not allow image and HTML files to be tracked, you will need to force all image and HTML files to be added to tracking history. The :file:`configure.sh` script contains the commands to add all the necessary files. The :file:`configure.sh` can be executed and the changes committed and pushed to GitHub:
 
+.. code-block:: none
 
+    ./configure.sh
+    git commit -am 'my commit message here'
+    git push origin gh-pages-update
 
+The final step in updating the website documentation is to `pull request`_ to the ``gh-pages`` branch of the OpenMOC repository on GitHub as described in :ref:`Development Workflow <work_flow>`.
```

Comment:
Perhaps note that the only reason we want PRs to "gh-pages" is to provide a transparent and publicly-available record of when the docs on the web have changed - and *not* to review the diff!